### DISCUSSION TITLE
Advice 📚🚀 Best libraries to power projects with Next.js? 🤔

### DISCUSSION THREAD

**Discussion by Miguel-ectil** [community/community#79126] [Programming Help]
### Body

Hey guys! 👋🏿 My name is Miguel Ectil, I'm a junior Front-End developer and I'm looking for advice from more experienced developers here in the community. I am currently immersed in a project with Next.js and would like to know which libraries you recommend or have used successfully in this context. Which ones can improve the development experience or offer useful features? I would be very grateful if you could share why you consider these libraries valuable. Thank you in advance for sharing your experiences and knowledge! 🌟

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by wodsuz** [comment]
Hi, It really depends on the project you are working, but here are my fav ones,

@next-auth  https://www.npmjs.com/package/next-auth its a great package I use when my sites require authentication or login logic.
@sendgrid to integrate with sendgrid and send mails automatically
@mongoose to interact with MongoDb
@stripe i prefer stripe in order to collect payments
@tailwind is for styling i believe it's way easier and straight forward than other css preprocessors

**Reply by Miguel-ectil** [reply]
Olá! Obrigado por sua contribuição e por compartilhar suas recomendações! Concordo que a escolha das bibliotecas depende em grande parte do contexto do projeto. Já usei alguns deles e concordo que são excelentes opções. é uma escolha que tenho ouvido muito ultimamente, então obrigado por destacá-la. Eu sempre optei por fazer autenticação de outra maneira, mas com as dicas que você compartilhou, estou animado para começar a implementar o next-Auth e explorar seus recursos. Vou dar uma olhada mais detalhada nisso. Obrigado pela colaboração! 😁🙏🏿👍🏿

**Comment by DarioChiappello** [comment]
if you have in mind UI packages, you can use https://nextui.org/ . Maybe you can use some components there. the documentation is great and you can have a preview of the components and the code

**Reply by Miguel-ectil** [reply]
Hmmm yes, I have been using shadcn ui as recommended by colleagues, but let me take a look at this nextui.

Thank you very, very much for the tip 😁🙏🏿👍🏿
